CONTRIBUTING.md

```diff
@@ -1,41 +1,78 @@
-# Contributing to eXist
-We welcome all contributions to eXist!
+# Contributing to eXist-db
+We welcome everyone to contribute to eXist-db. We will consider each individual contribution on its own merits.
+We strongly suggest that you join the [eXist-db Slack Channel](https://exist-db.slack.com), so that you can collaborate with the eXist-db community. It is often valuable to discuss a potential contribution before undertaking any work.
```

Comment:
s/Slack Channel/Slack workspace

Reply:
Thanks @joewiz I pushed a commit to address this.